index.html

```diff
@@ -2,27 +2,65 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <!-- UI Sheets -->
+    <link rel="stylesheet" type="text/css" href="css/materialize.css">
+    <link rel="stylesheet" type="text/css" href="css/fonts.css">
     <script type="text/javascript" src="js/sweetalert2.min.js"></script>
+    
+    <!-- Engine Scripts -->
     <script type="text/javascript" src="js/jquery.js"></script>
+    <script type="text/javascript" src="js/materialize.js"></script>	
     <script type="text/javascript" src="js/tiles.js"></script>
     <script type="text/javascript" src="js/levels.js"></script>
     <script type="text/javascript" src="js/player.js"></script>
     <script type="text/javascript" src="js/level.js"></script>
     <link rel="stylesheet" href="css/tiles.css">
     <title>Labyrinth</title>
   </head>
-  <body>
-    <div>
-        Player moved <span id="playerMoves">0</span> times.
+<body>
+<div class="navbar-fixed">
+	<nav>
+	  <div class="nav-wrapper">
+	    <a class="brand-logo center">Labyrinth</a>
+	    <ul class="right hide-on-med-and-down">
+	      <!-- Sample link -->
+	      <li><a href="howtoplay.html">How to Play</a></li> 
+	    </ul>
+	  </div>
+	</nav>
+</div>
+
+<!-- Data Card -->
+<div class="row">
+    <div class="col m4 s8">
+      <div class="card blue-grey darken-1">
+        <div class="card-content white-text">
+          <span class="card-title">Statistics</span>
+          <p>Player Moved : <span id="playerMoves">0</span> Times</p>
+        </div>
+        <div class="card-action">
+          <a href="http://github.com/yashkumarverma/">UI by Yash Kumar Verma</a>
```

Comment:
This is nothing for this pull request, but we could create a credits.html file where we could mention many more people for different tasks.